ml_perf/README.md

```diff
@@ -0,0 +1,114 @@
+# Preliminary
+This repo is a staging ground for the new [MLPerf](http://mlperf.org) reinforcement model.
+Eventually this code will replace the code in
+[this directory](http://github.com/mlperf/training/tree/master/reinforcement/tensorflow/minigo).
+
+# 1. Problem
+This task benchmarks on policy reinforcement learning for the 9x9 version of the boardgame Go.
+The model plays games against itself and uses these games to improve play.
+
+# 2. Directions
+### Steps to configure machine
+To setup the environment on Ubuntu 16.04 (16 CPUs, one P100, 100 GB disk), you can use these
+commands. This may vary on a different operating system or graphics card.
+
+```
+    # Clone repository
+    git clone https://github.com/tensorflow/minigo
+    # Note: This will eventually change to:
+    # git clone http://github.com/mlperf/training
+
+    # Install dependencies
+    apt-get install -y python3 python3-pip rsync git wget pkg-config zip g++ zlib1g-dev unzip
+
+    # Install python dependencies
+    pip3 install -r requirements.txt
+
+    # Install bazel
+    wget https://github.com/bazelbuild/bazel/releases/download/0.17.1/bazel-0.17.1-installer-linux-x86_64.sh
+    chmod +x bazel-0.17.1-installer-linux-x86_64.sh
+    ./bazel-0.17.1-installer-linux-x86_64.sh
+
+    # Compile TensorFlow
+    ./cc/configure_tensorflow.sh
```

Comment:
Yeah, I understand how much of a pain point this can be.

I'm going to attempt to rewrite `configure_tensorflow.sh` this week to pull in TensorFlow via Bazel's WORKSPACE mechanism (but still have it as a separate pre-build step rather than a direct dependency). So I'm going to hold off adding more documentation until that rewrite is done (or I fail and give up).